provisioning/templates-api/common.sh

```diff
@@ -33,6 +33,11 @@ kubectl apply -f ./kubernetes/deploy/namespace.yaml
 # Get etcd root password, if it is already present
 export ETCD_ROOT_PASSWORD=$(kubectl get secret --namespace "$NAMESPACE" templates-api-etcd -o jsonpath="{.data.etcd-root-password}" 2>/dev/null | base64 -d)
 
+# TEMPORARY:
+# Delete StatefulSet, keep pods, to resize the PVC disks.
+kubectl delete sts --namespace "$NAMESPACE" "templates-api-etcd"
+kubectl delete pvc --namespace "$NAMESPACE" "data-templates-api-etcd-0"
```

Comment:
Velkost disku sa neda pri StateFul set len tak zmenit + na Azure sa neda spravit resize pripojeneho disku.

Na Buffer Service sa velkost nemeni, a pre Templates API mozeme disky zmazat a znova vytvorit.
(V CI to s MiniKube preslo).

Reply:
podla mna by to mohlo ist aj bezvypadkovo, ale nenasiel som na rychlo ako na to..asi by tam musel byt nejaky operator co by ten resize rollout riadil. Kedze templates etcd nepotrebuje byt HA tak do toho zabrusovat nebudem :) 
btw s zmazanim sts sa nezmaze aj pvc? tj nasledny delete pvc potom vyfailuje..?